src/variety-common/MouseInput.js

```diff
@@ -3,6 +3,9 @@
 pzpr.classmgr.makeCommon({
 //---------------------------------------------------------
 MouseEvent:{
+	left : 'left',
+	right : 'right',
```

Comment:
hacky way to get the left mouse button to unshade for bag; there might be a nicer way to do this, considering the ui exposes an "invert mouse button" toggle